swan-lake/development-tutorials/test-document-the-code/test-ballerina-code/write-tests.md

```diff
@@ -169,6 +168,7 @@ The Ballerina test framework supports the following assertions, which help to ve
 
 ### Troubleshoot assertion failures when using `assertEquals`
 
+You can troubleshoot assertion failures of different values when using the `assertEquals` property as shown below.
```

Comment:
Since the the sub topics below this talk about the different types, I feel `types` is a better way of wording it.
```suggestion
You can troubleshoot assertion failures of different types when using the `assertEquals` function as shown below.
```